bot/utils/timestamp_validity.py

```diff
@@ -0,0 +1,29 @@
+from bot.constants import TIMESTAMP_MAKE_BOUNDARIES, TIMESTAMP_RELATIVE_BOUNDARIES
+from copy import deepcopy
+
+
+def make_data_validity(year: int, month: int, day: int, hour: int, minute: int, utc: int):
+    invalid_fields = []
+    boundaries = deepcopy(TIMESTAMP_MAKE_BOUNDARIES)
+    for item in boundaries:
+        input_value = locals()[item["name"]]
```

Comment:
This would probably be better to accept a dictionary as a parameter. Using locals for this isn't best practice.

Reply:
Now using dict